app/src/smmt/client.js

```diff
@@ -79,6 +79,7 @@ function getSmmtResponseSuperagent(marque, vin) {
 
 function vincheck(marque, vin) {
   logger = loggerFactory.create();
+  logger.debug(config, 'SMMT api configuration');
```

Comment:
I'd refrain from printing the config with the api key, even on debug level

Reply:
sorry, it should be removed before PR